src/indra_cogex/client/queries.py

```diff
@@ -952,3 +982,82 @@ def get_schema_graph(client: Neo4jClient) -> nx.MultiDiGraph:
             label=edge.type,
         )
     return graph
+
+
+# CCLE
+
+
+def is_gene_mutated(
+    gene: Tuple[str, str], cell_line: Tuple[str, str], *, client: Neo4jClient
+) -> bool:
+    """Return True if the gene is mutated in the given cell line.
+
+    Parameters
+    ----------
+    client :
+        The Neo4j client.
+    gene :
+        The gene to query.
+    cell_line :
+        The cell line to query.
+
+    Returns
+    -------
+    :
+        True if the gene is mutated in the given cell line.
+    """
+    return client.has_relation(
+        gene,
+        cell_line,
+        relation="mutated_in",
+        source_type="BioEntity",
+        target_type="BioEntity",
+    )
+
+
+# Indra DB
+
+
+def get_drugs_for_target(
+    target: Tuple[str, str], *, client: Neo4jClient
+) -> Iterable[Agent]:
+    """Return the drugs targetting the given protein."""
+    rels = client.get_source_relations(
+        target, "indra_rel", source_type="BioEntity", target_type="BioEntity"
+    )
+    drug_rels = [rel for rel in rels if _is_drug_relation(rel)]
+    drug_nodes = [(rel.source_ns, rel.source_id) for rel in drug_rels]
+    stmt_jsons = [json.loads(rel.data["stmt_json"]) for rel in drug_rels]
+    stmts = stmts_from_json(stmt_jsons)
+    drug_agents = [stmt.subj for stmt in stmts]
+    return {"nodes": drug_nodes, "agents": drug_agents}
```

Comment:
I find it a little bit odd that both nodes and agents are returned here. If the nodes are provided, they ought to represent the grounding of the drug such that 
```
stmt_jsons = [json.loads(rel.data["stmt_json"]) for rel in drug_rels]
stmts = stmts_from_json(stmt_jsons)
drug_agents = [stmt.subj for stmt in stmts]
```
should be redundant, I think, and could be replaced by outside code doing
```
get_standard_agent(node.data.name, {node.db_ns: node.db_id})
```
if constructing an Agent is needed.

Reply:
I agree that it makes more sense to return the nodes only at the query level. Two reasons why I implemented it this way here:
1) since we have to filter based on relation properties (Inhibition from "tas") and not on the node properties, we get relations back, so instead of full node objects we get relation.source_ns and relation.source_id (so no name and other node data_
2) agents are available here since we have statement json

I think the solution here would be to get access to the full node data when we only have a ns, id pair